android_trace/README.md

```diff
@@ -1,21 +1,21 @@
-<div align="center">
+<div align="center" class="rustdoc-hidden">
 
 # Android Trace
 
 </div>
 <!-- Close the <div> opened in lib.rs for rustdoc, which hides the above title -->
```

Comment:
You removed the comment from the other readme, so I guess it should be removed here too as the closing `div` was removed.

```suggestion
```